docs/en/ingest-management/security/mutual-tls.asciidoc

```diff
@@ -0,0 +1,200 @@
+
+[discrete]
+=== {agent} settings
+During {agent} installation on premise use the following options:
+
+[cols="1,1"]
+|===
+|`-certificate-authorities`
+|List of CA certificates that are trusted when {fleet-server} connects to {agent} 
+
+|`-elastic-agent-cert`
+|{agent} certificate to present to {fleet-server} during authentication
+
+|`-elastic-agent-cert-key`
+|{agent} certificate key to present to {fleet-server}
+|===
+
+[discrete]
+[[mutual-tls-on-premise-hosted-es]]
+== {fleet-server} on-premise and Hosted Elasticsearch Service
+
+In some scenarios you may want to deploy {fleet-server} on your own premises. In this case, you're able to provide your own certificates and certificate authority to facilitate mTLS between {fleet-server} and {agent}.
+
+However, as with the <<mutual-tls-cloud,{fleet-server} on {ecloud}>> use case, the data plane connections are terminated at Elastic's Cloud Proxy, which is not a multi-tenanted service and therefore can't provide per-user certificates.  
+
+image::images/mutual-tls-fs-onprem.png[Diagram of mutual TLS with Fleet Server on premise and hosted Elasticsearch Service deployment model]
+
+Similar to the {fleet-server} on {ecloud} use case, a secure proxy can be placed in such an environment to terminate the TLS connections and satisfy the mTLS requirements.
+
+image::images/mutual-tls-fs-onprem-proxy.png[Diagram of mutual TLS with Fleet Server on premise and hosted Elasticsearch Service deployment model with a proxy]
+
+[discrete]
+=== {agent} settings
+During {agent} installation on premise use the following options, similar to <<mutual-tls-on-premise,{agent} deployment on premises>>:
+
+[cols="1,1"]
+|===
+|`-certificate-authorities`
+|List of CA certificates that are trusted for when {fleet-server} connects to {agent} 
+
+|`-elastic-agent-cert`
+|{agent} certificate to present to {fleet-server} during authentication
+
+|`-elastic-agent-cert-key`
+|{agent} certificate key to present to {fleet-server}
+|===
+
+[discrete]
+=== {fleet-server} settings
+During {fleet-server} installation on-premise use the following options so that {fleet-server} can authenticate itself to the agent and then also to the secure proxy server:
+
+[cols="1,1"]
+|===
+|`-fleet-server-es-ca`
+|CA to use for the {es} connection, via secure proxy. This CA is used to authenticate the TLS connection from a secure proxy
+
+|`-certificate-authorities`
+|List of CA certificates that are trusted when {agent} connects to {fleet-server}
+
+|`-fleet-server-cert`
+|{fleet-server} certificate to present to {agents} during authentication
+
+|`-fleet-server-cert-key`
+|{fleet-server} certificate key to present to {agents}  
+|===
+
+[discrete]
+=== {fleet} settings
+
+This is the same as what's described for <<mutual-tls-on-premise,on premise deployments>>. The main difference is that you need to use certificates that are accepted by the secure proxy, as that is where the TLS connection will be terminated.
```

Comment:
[Suggestion]
Personally I'd rather focus more on the mTLS than the TLS termination itself. Also because you can have TLS termination without mTLS. Something in the line of:

```suggestion
This is the same as what's described for <<mutual-tls-on-premise,on premise deployments>>. The main difference is that you need to use certificates that are accepted by the secure proxy, as the mTLS is setup between the agent and the secure proxy.
```

Reply:
Changed to:

> This is the same as what's described for <<mutual-tls-on-premise,on premise deployments>>. The main difference is that you need to use certificates that are accepted by the secure proxy, as the mTLS is set up between the agent and the secure proxy.